### PR TITLE
add new columns to games table

### PR DESCRIPTION
**Pull request by cvalesares**
added the following columns to the games table for frontend consumption:
* game_time
* city
* state
* country

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -20,6 +20,9 @@
     t.integer "score"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.time "game_time"
```
Comment (chazsimons) [inline db/schema.rb:23]: My favorite part of reviewing code is seeing new stuff like this. I didn't realize Time was a column type, how cool!

File: db/migrate/20220108175912_addcountry_to_games.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class AddcountryToGames < ActiveRecord::Migration[5.2]
+  def change
```
Comment (chazsimons) [inline db/migrate/20220108175912_addcountry_to_games.rb:2]: Looks like we need to add the country still. An easy fix for sure.

**Comment by chazsimons** [comment]
Nice job using the rails migration methods to add new columns!
I left some comments but the big thing before approving is adding the country column.

File: db/schema.rb (hunk 1 above)
Comment (cvalesares) [reply db/schema.rb:23]: The only one I didn't double check in Postico is where I messed up 🥲

File: db/migrate/20220108175912_addcountry_to_games.rb (hunk 2 above)
Comment (cvalesares) [reply db/migrate/20220108175912_addcountry_to_games.rb:2]: The only one I didn't double check in Postico is where I messed up 🥲

**Comment by chazsimons** [comment]
Looking good!

**Pull request merged**